### ISSUE TITLE
tibble keep rownames

### ISSUE THREAD

**Issue by pratikunterwegs** [pratikunterwegs/tres-tidy-tutorial#28]
include rownames argument in `as_tibble` for `mtcars` example (section 2.1).
helps people who want to keep rownames when converting `matrix` to `tibble`

**Comment by richelbilderbeek** [comment]
I suggest not to show how to add row names when converting to a tibble, as the literature recommends to avoid doing so: I quote from [https://tibble.tidyverse.org/reference/rownames.html](https://tibble.tidyverse.org/reference/rownames.html), a Tidyverse page:

> While a tibble can [...].
> Generally, it is best to avoid row names, because they are basically a character column with different semantics than every other column.

I also know that in code I maintained that did use row names, I get a deprecation message.

**Comment by pratikunterwegs** [comment]
The suggested change is to show how to retain rownames which would otherwise be lost when using `as_tibble`, eg, from a matrix as a column in a tibble, using the `rownames ` argument to `as_tibble`.

**Comment by richelbilderbeek** [comment]
You are right. Silly me :man_facepalming: . One day, I can read properly :sunglasses: :+1:
